packages/gatsby-transformer-sharp/README.md

```diff
@@ -38,6 +38,28 @@ It recognizes files with the following extensions as images.
 
 Each image file is parsed into a node of type `ImageSharp`.
 
+## Configuration options
+
+`checkSupportedExtensions` [boolean][optional]
+
+Sharp only supports certain image formats (see Parsing algorithm) and hence throws a warning when you e.g. use a .gif in an `ImageSharp` query. You'll need to use `publicURL` instead. With this option you can disable the warning behavior.
```

Comment:
```suggestion
Sharp only supports certain image formats (see Parsing algorithm) and hence throws a warning when you e.g. use a .gif in an `ImageSharp` query. You'll need to use `publicURL` instead. With this option you can disable the warning behavior.
```
Where is the parsing algorithm? Can we link to it?

Reply:
It's the paragraph above of this section :)

Reply:
Haha of course it is. Maybe we can make it (see the Parsing algorithm section above)